packages/puppeteer/src/browser.ts

```diff
@@ -33,5 +33,8 @@ export function configureAxe(config?: Axe.Spec) {
       application: 'axe-puppeteer'
     }
   };
-  window.axe.configure(brandingConfig);
+  window.axe.configure({
+    allowedOrigins: ['<unsafe_all_origins>'],
+    ...brandingConfig
+  });
```

Comment:
What is the purpose of having `brandingConfig` hoisted up? Seems like this would be just fine:

```js
window.axe.configure({
  allowedOrigins: ['<unsafe_all_origins>'],
  branding: { application: 'axe-puppeteer' }
})
```

Another not a big deal/not worth changing thing btw 🤷